src/components/CreateAddress/AddressGenerator.jsx

```diff
@@ -6,6 +6,7 @@ import {
   scriptToHex,
   multisigRedeemScript,
   multisigWitnessScript,
+  validatePublicKey,
```

Comment:
Just a note that this PR will also require a version bump to `unchained-bitcoin` before it can be merged in.